esp/src/lws.config.js

```diff
@@ -1,4 +1,5 @@
-const debugServerIP = "192.168.99.103";
```

Comment:
Did you mean to check in this configuration? Probably not, due to no new paths / mappings being added.